pkgs/development/libraries/gdal/default.nix

```diff
@@ -166,7 +169,7 @@ stdenv.mkDerivation (finalAttrs: {
       giflib
       libjpeg
       json_c
-      lerc
+      lerc # TODO: Remove?
```

Comment:
Why remove it?

Reply:
Once LERC support works I want to check whether removing this changes anything. That is, once we change the rest of the derivation to fix LERC support, is this line redundant?

Reply:
GDAL should be built with lerc when it is found during configure phase. I believe dropping it would cause GDAL build without lerc support.